docs/table_design/Temporary_partition.md

```diff
@@ -0,0 +1,311 @@
+
+You can load data into one or more temporary partitions by using the `INSERT INTO` command, STREAM LOAD, or BROKER LOAD.
+
+### Load data by using `INSERT INTO` command
+
+Example:
+
+```SQL
+INSERT INTO site_access TEMPORARY PARTITION (tp1) VALUES ("2020-01-01",1,"ca","lily",4);
+INSERT INTO site_access TEMPORARY PARTITION (tp2) SELECT * FROM site_access_copy PARTITION p2;
+INSERT INTO site_access TEMPORARY PARTITION (tp3, tp4,...) SELECT * FROM site_access_copy PARTITION (p3, p4,...);
+```
+
+For detailed information such as the syntax and parameters descriptions, see [INSERT INTO](../sql-reference/sql-statements/data-manipulation/insert.md).
+
+### Load data by using STREAM LOAD
+
+Example:
+
+```undefined
+curl --location-trusted -u root: -H "label:123" -H "temporary_partitions: tp1, tp2, ..." -T testData \
+    http://host:port/api/example_db/site_access/_stream_load    
+```
+
+For detailed information such as the syntax and parameters descriptions, see [STREAM LOAD](..sql-reference/sql-statements/data-manipulation/STREAM%20LOAD.md).
+
+### Load data by using BROKER LOAD
+
+Example:
+
+```SQL
+LOAD LABEL example_db.label1
+(
+    DATA INFILE("hdfs://hdfs_host:hdfs_port/user/starrocks/data/input/file")
+    INTO TABLE my_table
+    TEMPORARY PARTITION (tp1, tp2, ...)
+    ...
+)
+WITH BROKER
+(
+    StorageCredentialParams
+);
+LOAD LABEL example_db.label1
+(
+    DATA INFILE("hdfs://hdfs_host:hdfs_port/user/starrocks/data/input/file")
+    INTO TABLE my_table
+    TEMPORARY PARTITION (tp1, tp2, ...)
+    ...
+)
+WITH BROKER "mybroker"
+(
+    "username" = "hdfs_username",
+    "password" = "hdfs_password"
+);
+```
+
+For detailed information such as the syntax and parameters descriptions, see [Broker Load](../sql-reference/sql-statements/data-manipulation/BROKER%20LOAD.md).
+
+### Load data by using ROUTINE LOAD
+
+Example:
+
+```SQL
+CREATE ROUTINE LOAD example_db.site_access ON example_tbl
+COLUMNS(col, col2,...),
+TEMPORARY PARTITIONS(tp1, tp2, ...)
+FROM KAFKA
+(
+    "kafka_broker_list" ="<kafka_broker1_ip>:<kafka_broker1_port>,<kafka_broker2_ip>:<kafka_broker2_port>",
+    "kafka_topic" = "ordertest"
+);
+```
+
+For detailed information such as the syntax and parameters descriptions, see [CREATE ROUTINE LOAD](../sql-reference/sql-statements/data-manipulation/CREATE%20ROUTINE%20LOAD.md).
+
+## Query data in temporary partitions
+
+You can use the [SELECT](../sql-reference/sql-statements/data-manipulation/SELECT.md) statement to query data in specified temporary partitions.
+
+```SQL
+SELECT * FROM
+site_access TEMPORARY PARTITION (tp1);
+
+SELECT * FROM
+site_access TEMPORARY PARTITION (tp1, tp2, ...);
+
+SELECT event_day,site_id,pv FROM
+site_access TEMPORARY PARTITION (tp1, tp2, ...);
+```
+
+You can use the JOIN clause to query data in temporary partitions from two tables.
+
+```SQL
+SELECT * FROM
+site_access TEMPORARY PARTITION (tp1, tp2, ...)
+JOIN
+site_access_copy TEMPORARY PARTITION (tp1, tp2, ...)
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/fc122cbb-6a45-40df-829f-41cd26d75c1e)
should be "(<temporary_partition1_name>[, <temporary_partition2_name>, ...)